core/build.gradle

```
@@ -21,6 +23,8 @@ android {
 
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
+    implementation "androidx.core:core-ktx:1.0.1"
```

latest core version is 1.3.1
```suggestion
    implementation "androidx.core:core-ktx:1.3.1"
```

---

I had these updated in the following PR to this one but I'll commit these cause why the hell not